.github/workflows/update_file.yaml

```diff
@@ -0,0 +1,38 @@
+name: Update files
```

Comment:
**suggestion:** 

It will be nice if you can explain the whole workflow purpose here.